plugins/Kaleidoscope-MouseKeys/src/kaleidoscope/plugin/MouseKeysConfig.cpp

```diff
@@ -49,41 +46,33 @@ EventHandlerResult MouseKeysConfig::onSetup() {
 }
 
 // -----------------------------------------------------------------------------
-EventHandlerResult MouseKeysConfig::onFocusEvent(const char *command) {
-  // If the focus command is a request for help, provide the list of valid
-  // commands.
-  if (::Focus.handleHelp(command, PSTR("mousekeys.scroll_interval\n"
-                                       "mousekeys.init_speed\n"
-                                       "mousekeys.base_speed\n"
-                                       "mousekeys.accel_duration")))
-    return EventHandlerResult::OK;
-
-  // The length of the string `mousekeys.`:
-  constexpr uint8_t base_cmd_len = 10;
-
-  // If this is not a MouseKeys command, do nothing.
-  if (strncmp_P(command, PSTR("mousekeys."), base_cmd_len) != 0)
-    return EventHandlerResult::OK;
-  // Otherwise, advance the pointer to the subcommand.
-  command += base_cmd_len;
-
-  enum Command : uint8_t {
```

Comment:
What's your reason for not wanting a scoped `enum` here?  I object to this change, because it both makes things more ambiguous when reading the code, and removes the specified type that forces it to be a one-byte value instead of two (thus saving space on smaller MCUs).

Reply:
I changed it for terser code as I adopted the syntax used in other plugins, but will revert this